smithy-model/src/main/java/software/amazon/smithy/model/validation/validators/PrivateAccessValidator.java

```diff
@@ -38,26 +35,26 @@
 
     @Override
     public List<ValidationEvent> validate(Model model) {
-        Set<Shape> privateShapes = model.getShapesWithTrait(PrivateTrait.class);
-        NeighborProvider provider = NeighborProviderIndex.of(model).getProvider();
-        return model.shapes()
-                .filter(shape -> !(shape instanceof SimpleShape))
```

Comment:
I don't see this filter carried forward, but assume that's intentional given the `*` selector on `@private` - should this come with a test?